.schema/config.schema.json

```diff
@@ -4,6 +4,11 @@
   "title": "ORY Kratos Configuration",
   "type": "object",
   "definitions": {
+    "version": {
+      "type": "string",
+      "description": "SemVer according to https://semver.org/ prefixed with `v` as in our releases.",
+      "pattern": "^v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
This won't work with `latest` or generally builds that have not been built as part of the CI build pipeline, right?

Reply:
yes but it would not make sense to have `version: latest` in the config anyway. In case of latest/master/... we would ignore the key completly

Reply:
That would be the idea!